src/test/rustdoc/issue-12834.rs

```diff
@@ -15,7 +15,7 @@
 // @has issue_12834/fn.foo.html
 // @has - //pre 'a + b '
 
-/// ```
+/// ```text
```

Comment:
As the comment above says, this test is making sure rustdoc doesn't fail even if syntax highlighting fails so this change makes this test useless. Was there a reason for this change?